program/src/oracle/pd.h

```diff
@@ -152,21 +152,21 @@ static void pd_sub( pd_t *r, const pd_t *n1, const pd_t *n2, const int64_t *p )
   pd_scale( r );
 }
 
-static int pd_lt( const pd_t *n1, const pd_t *n2, const int64_t *p )
+[[maybe_unused]] static int pd_lt( const pd_t *n1, const pd_t *n2, const int64_t *p )
 {
   pd_t r[1];
   pd_sub( r, n1, n2, p );
   return r->v_ < 0L;
 }
 
-static int pd_gt( const pd_t *n1, const pd_t *n2, const int64_t *p )
+[[maybe_unused]] static int pd_gt( const pd_t *n1, const pd_t *n2, const int64_t *p )
 {
   pd_t r[1];
   pd_sub( r, n1, n2, p );
   return r->v_ > 0L;
 }
 
-static void pd_sqrt( pd_t *r, pd_t *val, const int64_t *f )
+[[maybe_unused]] static void pd_sqrt( pd_t *r, pd_t *val, const int64_t *f )
```

Comment:
We should fix these either by marking them as `inline` or moving the function definitions to a `pd.c` file. I'll make that change separately though; this is fine for now.

Reply:
Fixed in https://github.com/pyth-network/pyth-client/pull/94